install/homarr-install.sh

```diff
@@ -31,12 +31,15 @@ msg_ok "Set up Node.js Repository"
 msg_info "Installing Node.js/Yarn"
 $STD apt-get update
 $STD apt-get install -y nodejs
-$STD npm install -g npm@latest
 $STD npm install -g yarn
 msg_ok "Installed Node.js/Yarn"
 
 msg_info "Installing Homarr (Patience)"
-$STD git clone -b dev https://github.com/ajnart/homarr.git /opt/homarr
+cd /opt
```

Comment:
```suggestion
```

Reply:
why?

Reply:
It is not needed

Reply:
There is `cd /opt/homarr` on line 51 before install